rackcity/views/it_model_views.py

```diff
@@ -267,7 +267,8 @@ def records_are_identical(existing_data, new_data):
         if key not in new_keys and existing_data[key] is not None and key != 'id':
             return False
         if key in new_keys and new_data[key] != existing_data[key]:
-            return False
+            if not (isinstance(existing_data[key], int) and int(new_data[key]) == existing_data[key]):
```

Comment:
should this be `if not (isinstance(new_data[key], int)` ? because we're worried about data from the front end being sent as strings? 

So if `new_data[key] = "8"` and `existing_data[key] = 8`, that's a match
But if `new_data[key] = "8" and `existing_data[key] = 9, that's not a match

Reply:
scratch that never mind, I was reading the `(` wrong 🥴
thoughts on splitting this into multiple lines so it's more readable?

Reply:
Yeppo will try to make more readable